src/main/resources/db/migration/V6.1.0.8.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE IF NOT EXISTS makedatacountprocessstate (
```

Comment:
is this needed? generally we have let JPA create tables for new entities)

Reply:
removed the sql file and added index to MakeDataCountProcessState.java